src/main/java/bio/terra/cloudfunctions/proto/ProtoApp.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cloudfunctions.proto;
+
+import bio.terra.cloudevents.CloudStorageEventType;
+import bio.terra.cloudevents.GCSEvent;
+import bio.terra.cloudfunctions.common.App;
+import java.util.logging.Logger;
+
+/** This class is a subclass of App with a custom implementation of the process() logic. */
+public class ProtoApp extends App {
+  private static final Logger logger = Logger.getLogger(ProtoApp.class.getName());
+
+  public ProtoApp(CloudStorageEventType cloudStorageEventType, GCSEvent cloudStorageEvent) {
+    super(cloudStorageEventType, cloudStorageEvent);
+  }
+
+  // Business logic
+  @Override
+  public void process() throws Exception {
+    if (CloudStorageEventType.GOOGLE_STORAGE_OBJECT_FINALIZE
+        .getCode()
+        .equals(getCloudStorageEventType().getCode())) {
+      logger.info(
+          String.format(
+              "Received %s event from bucket %s of content type %s",
+              getCloudStorageEventType().getDesc(),
+              getCloudStorageEvent().getBucket(),
+              getCloudStorageEvent().getContentType()));
```

Comment:
It looks like this function just logs the event data?  Do we want to change it to handle test runner's case?  If this is just an example, then I'd suggest removing the code and adding some code snippets to the documentation.

Reply:
The main purpose of this `ProtoFunc` / `ProtoApp` is so that everybody feels comfortable with `CloudStorageEventHarness` . Once everyone reviews and comfortable with it, I will use it to build the Test Runner case.

Reply:
Ok, I will look at `CloudStorageEventHarness` closely and see what sort of feedback I can offer there.  With this PR I think we should strive not to merge anything into `main` that needs immediate follow on review.